### ISSUE TITLE
Sequential class seems to not present an equivalent attribute to "trainable_variables"

### ISSUE THREAD

**Issue by BananaFructa** [SciSharp/Keras.NET#171]
I am trying to get the gradients of a model using tf.GradientTape().gradient() from Tensorflow.NET but it seems that the Sequential model doesn't present any "trainable_variables" attribute unlike in python. Looked into the documentation and it also doesn't seem to exist.
